reference_controllers/rabbitmq_controller_unverified/src/role.rs

```diff
@@ -0,0 +1,46 @@
+use crate::rabbitmqcluster_types::RabbitmqCluster;
+use k8s_openapi::api::apps::v1 as appsv1;
+use k8s_openapi::api::core::v1 as corev1;
+use k8s_openapi::api::rbac::v1 as rbacv1;
+use k8s_openapi::apimachinery::pkg::api::resource::Quantity;
+use k8s_openapi::apimachinery::pkg::apis::meta::v1 as metav1;
+use k8s_openapi::apimachinery::pkg::util::intstr::IntOrString;
+use kube::{
+    api::{Api, ListParams, PostParams},
+    runtime::controller::{Action, Controller},
+    Client, CustomResourceExt,
+};
+use kube_client::{self, client};
+use kube_core::{self, Resource};
+use std::collections::BTreeMap;
+
+pub fn role_build(rabbitmq: &RabbitmqCluster) -> rbacv1::Role {
+    let name_new = rabbitmq.metadata.name.clone().unwrap() + "-peer-discorvery";
```

Comment:
The variable name `name_new` is confusing here. Where is `name_old` then?  Why not just call it `name`?